roles/debian/hypervisor/tasks/main.yml

```diff
@@ -272,3 +272,29 @@
     name: ptp_vsock.service
     enabled: yes
     state: started
+
+- block:
+  - name: make sure conntrackd package is installed
+    ansible.builtin.apt:
```

Comment:
I would prefer checking that the systemd service `contrackd` exists, rather than the apt package.

Reply:
same answer as before, this is a workaround to install the package on the fly, while the change is not merged

Reply:
Okay, but please don't forget to remove this apt part later and to move that in the network playbook.